languageServer/src/ITextDocumentWrapper.ts

```diff
@@ -0,0 +1,7 @@
+import { Position } from 'vscode-languageserver-textdocument'
+import { Document } from 'yaml'
+
+export interface ITextDocumentWrapper {
```

Comment:
We're not prefixing our interfaces with `I` anywhere else. I think it would be best to stick to the current format (no prefix).

Reply:
I do use the I prefix in the extension in at least one place where multiple things implement the interface (`ICli`)

Reply:
Yes, I looked it up beforehand to see if there were precedents 😁

But It's just a cosmetic quirk from me, I don't know if there is any real benefit from it.

I can take them out for consistency